src/index.ts

```diff
@@ -1799,29 +1800,20 @@ async function claimLinkGasless({
 	link,
 	recipientAddress,
 	APIKey,
-	baseUrl = 'https://api.peanut.to/claim',
+	baseUrl = 'https://api.peanut.to/claim-v2',
 }: interfaces.IClaimLinkGaslessParams) {
 	config.verbose && console.log('claiming link through Peanut API...')
-	config.verbose &&
-		console.log('link: ', link, ' recipientAddress: ', recipientAddress, ' apiKey: ', APIKey, ' url: ', baseUrl)
 	const payload = await createClaimPayload(link, recipientAddress)
 	config.verbose && console.log('payload: ', payload)
-	if (baseUrl == 'local') {
-		config.verbose && console.log('using local api')
-		baseUrl = 'http://127.0.0.1:5001/claim'
-	}
 
 	const headers = {
 		'Content-Type': 'application/json',
 	}
-
-	// TODO: change the API to accept just pure claimParams
-	// This is needed to work with both v4.2 and older versions
 	const body = {
 		claimParams: payload.claimParams,
 		chain: payload.chainId,
 		version: payload.contractVersion,
-		api_key: APIKey,
+		apiKey: APIKey,
```

Comment:
Didnt you change this in the API already?

Reply:
probably previously there was `api_key` since the previous API code was made in python and in python variables should be named in snake case, so `api_key` naming was was in line with the rest of the python code.

Now we have both the client and the server implemented in js / ts and we use camel case for variable naming on both sides, so I believe we should switch to camel case naming in new endpoints.